scripts/overlap.py

```diff
@@ -1,9 +1,11 @@
 """Computes the overlap of molecules between two datasets."""
 
 from argparse import ArgumentParser, Namespace
+from os.path import dirname, realpath
 import csv
 import sys
-sys.path.append('../')
+
+sys.path.append(dirname(dirname(realpath(__file__))))
```

Comment:
I like that you shortened this by importing `dirname` and `realpath`, but just for consistency do you mind doing it the same way as in the other files? See below.

```suggestion
sys.path.append(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
```